docs/gateway-diagram-internal-file.dot

```diff
@@ -0,0 +1,31 @@
+digraph {
+  subgraph external_user {
+    "User" -> "downloads.acme.com" [label = "curl -LO https://downloads.acme.com/files/2.2.1/amd64/rocket-skates-amd64-2.2.1.tar.gz"];
+    "downloads.acme.com" -> "gw.scarf.sh" [label=""];
+  }
+
+  subgraph external_scarf {
+    "Maintainer" -> "Scarf Dashboard";
+    "Scarf Analytics" -> "Scarf Dashboard" [label="Data is processed and insights are made available to the maintainer"];
+    "Maintainer" -> "DNS Provider" -> "downloads.acme.com" [style=dashed];
+    "downloads.acme.com" -> "gw.scarf.sh" [style=dashed label="Maintainer set a CNAME from their own domain to Scarf"]
+  }
+
+  subgraph cluster_internal_scarf {
+    label = "Scarf Infrastructure"
+    "gw.scarf.sh" -> "Scarf Gateway" [label="Request hits Scarf gateway"];
+    "Scarf Gateway" -> "Scarf Configuration";
+    "Scarf Configuration" -> "Scarf Gateway";
+    "Scarf Gateway" -> "Scarf Analytics" [label="Requests aggregated to be asynchronously processed"];
+    "Scarf Dashboard" -> "Scarf Configuration" [style=dashed label="Maintainers configure the Scarf Gateway to redirect to the correct hosting provider"];
```

Comment:
We want to be consistent with use of _The_ Scarf Gateway or just Scarf Gateway. Over time, I'm leaning towards dropping "the", but let's use the same thing across if we are changing it.

Reply:
I agree on consistency. diagrams seem weird with "the" in them IMHO

Reply:
We don't use The Scarf Server anywhere else. I agree on consistent. Was trying to do that everywhere in this doc. Looking forward to a tech writer handling it.